From:  Breno Leitao <breno.leitao@gmail.com>
Date: Mon, 28 Nov 2016 15:12:29 -0500
Subject: Revert compiling with O2 on ppc64el due to a known bug

Due to GCC bug yadifa could not be compiled with O3 on ppcel.
This was fixed upstream in GCC7 and was backported to stretch.
Hence revert the original patch now included upstream

References:
 https://gcc.gnu.org/bugzilla/show_bug.cgi?id=71873
 https://gcc.gnu.org/bugzilla/show_bug.cgi?id=78543

--- a/m4/eurid.m4
+++ b/m4/eurid.m4
@@ -298,12 +298,6 @@
 		CPU_UNKNOWN=0
 		cpu_intel_compatible=1
 		;;
-    ppc64le)
-
-        dnl https://github.com/asciiprod/yadifa/blob/master/debian/patches/fix-ppc64el_ftbfs.patch
-
-        cpu_power_compatible=1
-        ;;
 	*)
 		;;
 esac
@@ -632,14 +626,6 @@
 		fi
 	fi
 
-    dnl Move to O2 due to https://gcc.gnu.org/bugzilla/show_bug.cgi?id=78543
-    dnl https://github.com/asciiprod/yadifa/blob/master/debian/patches/fix-ppc64el_ftbfs.patch
-
-    if [[ $cpu_power_compatible -eq 1 ]]
-    then
-        CCOPTIMISATIONFLAGS=-O2
-    fi
-
     AM_CONDITIONAL([USES_ICC], [false])
     AM_CONDITIONAL([USES_GCC], [true])
     AM_CONDITIONAL([USES_CLANG], [false])
